lib/ae_mdw/activities.ex

```diff
@@ -333,6 +334,35 @@ defmodule AeMdw.Activities do
     end
   end
 
+  defp build_swaps_stream(state, direction, account_pk, txi_scope, txi_cursor) do
+    key_boundary =
+      case txi_scope do
+        nil ->
+          {
+            {account_pk, 0, Util.min_int(), nil},
+            {account_pk, nil, Util.max_int(), nil}
```

Comment:
All keys are going to be included between `0` and ` nil`, but we might as well use `Util.min_int()` to provide meaning to it:
```
            {account_pk, Util.min_int(), nil, nil},
            {account_pk, Util.max_int(), nil, nil}
```